simple-weather/simple-weather/Controller/ViewController.swift

```diff
@@ -40,9 +42,14 @@ class ViewController: UIViewController, CLLocationManagerDelegate {
         dateLabel.text = "Today, \(myDate)"
     }
     
+    func cellDelegate() {
+        tableView.delegate = self
```

Comment:
Remove delegate, your application does not need that. `DataSource` is responsible for data - number of rows, number of columns, row data, headers and footers, heights for elements etc. `Delegate` simply passes actions - tapped cell, swiped on cell etc.